clients/hadoopfs/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.lakefs</groupId>
+    <artifactId>hadoopfs</artifactId>
+    <packaging>jar</packaging>
+    <name>hadoopfs</name>
+    <version>0.1.0</version>
+    <url>https://github.com/treeverse/lakeFS/tree/master/clients/java</url>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
```

Comment:
It looks like we should use 
    <configuration>
        <source>1.8</source>
        <target>1.8</target>
      </configuration>

see for a reference http://tutorials.jenkov.com/maven/java-compiler.html